cmd/mafToFa/mafToFa_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"os"
+	"testing"
+)
+
+var mafToFaTests = []struct {
+	mafFile          string
+	refFile          string
+	speciesFile      string
+	outFile_expected string
+	noMask           bool
+}{
+	{"testdata/test1.maf", "testdata/test.ref.fa", "testdata/test.species.list", "testdata/test.mfa", false},
+	{"testdata/test2.maf", "testdata/test.ref.fa", "testdata/test.species.list", "testdata/test.mfa", true},
+}
+
+func TestMafToFa(t *testing.T) {
+	var err error
+	for _, v := range mafToFaTests {
+		mafToFa(v.mafFile, v.refFile, v.speciesFile, "outFile_tmp.mfa", v.noMask)
```

Comment:
pinging off Riley's comment about the .mfa   I think this tmp outfile will also need it's extension changed to .fa